infra/terraform/test-org/github/protection.tf

```diff
@@ -142,14 +142,14 @@ module "codeowners_gcp" {
 }
 
 module "lint_yaml_tgm" {
```

Comment:
Would be ideal to rename these and move the state: `terraform state mv module.lint_yaml_tgm module.workflow_files`

Reply:
Yeah we could use moved blocks. I will test it out in a follow up since it is non blocking.